docs/draft/keycard.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Keyacrd? (have no idea how to call it)
+version: 0.0.0
+status: Draft
+authors: Roman Volosovskyi <roman@status.im>
+redirect_from:
+  - /keycard.html
+---
+
+## Table of Contents
+
+1. [Abstract](#abstract)
+2. [Definitions](#definitions)
+3. [Multiaccount creation](#Multiaccount-creationrestoring)
+4. [Multiaccount restoring via pairing](#Multiaccount-restoring-via-pairing)
+5. [Multiaccount unlocking](#Multiaccount-unlocking)
+6. [Where are the keys stored?](#Where-are-the-keys-stored)
+
+## Abstract
+
+In this specification, we describe how Satus communicates with Keycard to create, store and use multiaccount.
+
+## Definitions
+
+
+| Term | Description |
+| -------- | ------------------ |
+|      |      |
+
+
+## Multiaccount creation/restoring
```

Comment:
Brief intro on multiaccount and why one wants to do this

Reply:
> why one wants to do this
Why one wants to do what, to create it or to use multiaccs in general?

Reply:
If the heading is describing creation/restoring, then that. So presumably something like "When setting up a new account a multiaccount is generated... A multiaccount is.. [inline or link to spec]... When this is created here is what happens... // when you re-initialize an account you might need to restore it..."

Reply:
thanks @rajeevgopalakrishna